basic-sql-commands.md

```diff
@@ -40,8 +40,8 @@ SHOW DATABASES;
 +----------+
 | Database |
 +----------+
-| db1      |
-| system   |
+| "db1"    |
```

Comment:
Please file an issue against the main repo to remove the quotes - at least from type names, column names, table names from the . String values are debatable, we can discuss in that issue. FWIW `psql` does not quote any of these (even when they have spaces).